### DISCUSSION TITLE
Can update data in component

### DISCUSSION THREAD

**Discussion by nguyenthuong1304** [livewire/livewire#3558] [1. Help]
<img width="679" alt="Screen Shot 2021-08-22 at 2 39 08 PM" src="https://user-images.githubusercontent.com/50571214/130346662-490e4b89-99ce-427c-8c89-6892c1e67e03.png">
<img width="675" alt="Screen Shot 2021-08-22 at 2 39 22 PM" src="https://user-images.githubusercontent.com/50571214/130346667-c89a6d31-5a28-4462-a39b-06e896de31d5.png">
<img width="767" alt="Screen Shot 2021-08-22 at 2 39 29 PM" src="https://user-images.githubusercontent.com/50571214/130346668-ae016812-a6e8-48ba-a59e-d2a71a68b599.png">

help me, why it not update data ?

**Comment by joshhanley** [comment]
@nguyenthuong1304 can you please explain in detail what the issue is and what data is not updating, and what you expect to be happening?

Thanks!
